app_flutter/lib/main.dart

```diff
@@ -9,14 +9,18 @@ import 'build_dashboard_page.dart';
 import 'index_page.dart';
 import 'service/google_authentication.dart';
 
-void main() => runApp(MyApp());
+void main() => runApp(const MyApp());
 
-final ThemeData theme = ThemeData(
-  appBarTheme: const AppBarTheme(color: Colors.green),
-  primarySwatch: Colors.blue,
+final ThemeData theme = ThemeData.from(
+  colorScheme: ColorScheme.light(
```

Comment:
Dark mode dashboard? 👀

Reply:
We can add support for dark mode if you want, it's pretty easy these days. :-)